qiskit/primitives/base/base_sampler.py

```diff
@@ -200,3 +258,38 @@ def parameters(self) -> tuple[ParameterView, ...]:
             List of the parameters in each quantum circuit.
         """
         return tuple(self._parameters)
+
+
+BaseSampler = BaseSamplerV1
```

Comment:
@jakelishman Is this the sort of thing we should have a `from __future__ import <what?>` so that `BaseSampler` gets set to BaseSamplerV2 instead? And that if you import BaseSampler without this import it raises a deprecation warning of some sort using the same trick you used in the other PRs and returns BaseSamplerV1 for it in those cases?